### ISSUE TITLE
Space indent size is not working in VSCode version 1.74

### ISSUE THREAD

**Issue by mazeneko** [editorconfig/editorconfig-vscode#344]
There is already a duplicate issue, but the title and description did not express the essence of the issue, so a new issue was created.

- https://github.com/editorconfig/editorconfig-vscode/issues/341
- https://github.com/editorconfig/editorconfig-vscode/issues/343

Also, the content of this issue was written based on this comment.

- https://github.com/editorconfig/editorconfig-vscode/issues/341#issuecomment-1344255324

## Task

- [x] I have a question that is specific to this extension; thus, inappropriate for the main [EditorConfig issue tracker](https://github.com/editorconfig/editorconfig/issues).
- [x] I tried running `code --disable-extensions` and the issue did **NOT** present itself.

## Issue

|             | Visual Studio Code | editorconfig-vscode |
|-------------|--------------------|---------------------|
| **Version** | `1.74.0`            | `0.16.4`             |

### Root `.editorconfig` File

```ini
root = true

[*]
indent_style = space
indent_size = 2
```

| Visual Studio Code Setting     | Default | User    | Workspace |
|--------------------------------|---------|---------|-----------|
| `editor.insertSpaces`          | `true`  | `____`  | `____`    |
| `editor.tabSize`               | `4`     | `_`     | `_`       |
| `editor.trimAutoWhitespace`    | `true`  | `____`  | `____`    |
| `files.autoSave`               | `"off"` | `"___"` | `"___"`   |
| `files.insertFinalNewline`     | `false` | `_____` | `_____`   |
| `files.trimTrailingWhitespace` | `false` | `_____` | `_____`   |

### Expected behavior

```ini
indent_style = space
indent_size = 2
```

### Actual behavior

```ini
indent_style = space
indent_size = 4 #This means the current value of vscode
```

### Reason

vscode has been updated to separate `tabSize` and `indentSize` in [1.74](https://code.visualstudio.com/updates/v1_74#_new-indent-size-setting),
but editorconfig-vscode applies the `indent_size` option in `.editorconfig` only to vscode's `tabSize`,
and `indentSize` is This seems to be due to the fact that it has not been changed.

In the actual code, only the tabSize is generated here, 

https://github.com/editorconfig/editorconfig-vscode/blob/b86ebd4e9062c76e7145bcefc1a17cf2e6cc1acc/src/api.ts#L160-L172

and the vscode setting is changed here.

https://github.com/editorconfig/editorconfig-vscode/blob/b86ebd4e9062c76e7145bcefc1a17cf2e6cc1acc/src/api.ts#L54

Also, I added the `indentSize` property in the debugger on this line and run it, and it was set as expected.

However, there is no `indentSize` in the TextEditorOptions type to actually change the code.

**Comment by mazeneko** [comment]
The absence of indentSize in the TextEditorOptions type is discussed in this issue.

- https://github.com/microsoft/vscode/issues/168836

**Comment by ThexXTURBOXx** [comment]
This seems to be fixed in 1.74.1 now.

**Comment by STOWouters** [comment]
> This seems to be fixed in 1.74.1 now.

I concur. I reproduced the linked issues with 1.74.0 and 1.74.1 versions. The issue is present in 1.74.0 but is fixed in 1.74.1.
